io/module_write_netcdf.F90

```diff
@@ -19,11 +28,21 @@ module module_write_netcdf
   private
   public write_netcdf
 
-  logical :: par
-
-  contains
-
-!----------------------------------------------------------------------------------------
+  logical :: par !< True if parallel I/O should be used.
+
+contains
+
+  !> Write netCDF file.
+  !>
+  !> @param[in] wrtfb ESMF write field bundle
+  !> @param[in] filename NetCDF filename.
+  !> @param[in] use_parallel_netcdf True if parallel I/O should be used.
+  !> @param[in] mpi_comm MPI communicator for parallel I/O.
+  !> @param[in] mype MPI rank
+  !> @param[in] grid_id Output grid identifier
+  !> @param[out] rc Return code: return code
```

Comment:
@DusanJovic-NOAA what is the rc for successful completion of the function? 0?

It seems that sometimes the rc is an ESMF error code, and other times a more local error code. Ideally we would like all the possible return codes here, but if not, then at least let's say what code is returned for success.

Reply:
Ok. Thanks.

Reply:
rc is equal to ESMF_SUCCESS if there are no errors. The value of ESMF_SUCCESS is 0.

Reply:
The list of all possible return codes is long.

https://earthsystemmodeling.org/docs/release/latest/ESMF_refdoc/node9.html#SECTION09030000000000000000

Reply:
OK, I will change it to:

return code - 0 for success, ESMF error code otherwise.

Reply:
OK, I have made that change.